src/Config/ConfigPanel.ts

```diff
@@ -0,0 +1,143 @@
+import * as vscode from 'vscode';
+import {getNonce} from './GetNonce';
+
+
+export class ConfigPanel {
+  /**
+   * Track the current panel. Only allow a single panel to exist at a time.
+   */
+  public static currentPanel: ConfigPanel|undefined;
+
+  public static readonly viewType = 'one-vscode';
+
+  private readonly _panel: vscode.WebviewPanel;
+  private readonly _extensionUri: vscode.Uri;
+  private _disposables: vscode.Disposable[] = [];
+
+  public static createOrShow(extensionUri: vscode.Uri) {
+    const column =
+        vscode.window.activeTextEditor ? vscode.window.activeTextEditor.viewColumn : undefined;
+
+    // If we already have a panel, show it.
+    if (ConfigPanel.currentPanel) {
+      ConfigPanel.currentPanel._panel.reveal(column);
+      ConfigPanel.currentPanel._update();
+      return;
+    }
+
+    // Otherwise, create a new panel.
+    const panel = vscode.window.createWebviewPanel(
+        ConfigPanel.viewType, 'ConfigurationSettings', column || vscode.ViewColumn.One, {
+          // Enable javascript in the webview
+          enableScripts: true,
+
```

Comment:
Is this empty line intentional ?

Reply:
empty line will be fulfilled later, but i don't think this is necessary now. so i fix it